include/boost/date_time/time_duration.hpp

```diff
@@ -29,8 +29,8 @@ namespace date_time {
       This design allows the subclass duration types to provide custom
       construction policies or other custom features not provided here.
 
-      @tparam T The subclass type
-      @tparam rep_type The time resolution traits for this duration type.
+      @param T The subclass type
```

Comment:
hmm I believe these where just change to tparam in another pull request...

Reply:
They were, so this should change to tparam.